### ISSUE TITLE
Drag and drop text copy

### ISSUE THREAD

**Issue by ghost** [tootsuite/mastodon#5756]
Thanks a ton from an average user for all the work done by everyone here! I love Mastodon.
I am just missing a drag and drop feature to push text in the "tooting area".
I am unfortunately unable to suggest any useful code. 
Thanks in advance for any answer.

**Comment by MMSadeghi** [comment]
what should be done about this issue?

**Comment by tmm576** [comment]
Got a PR up. Let me know what you think. Also, though not part of this PR, I also noticed drag and drop file uploads are limited to one file. Is this behavior by design?

**Comment by Cassolotl** [comment]
> I also noticed drag and drop file uploads are limited to one file. Is this behavior by design?

*Feature request: Ability to upload more than one image at once #5556* might be relevant?
